src/compiler/sys.ts

```diff
@@ -1,5 +1,9 @@
 /// <reference path="core.ts"/>
 
+// Java host
+declare var arguments: string[];
+const globalArguments = arguments;

```

Comment:
Rather than doing this, why not make `getJavaSystem` take an `commandLineArgs` parameter?